src/server/lib/__tests__/headers.test.ts

```diff
@@ -28,7 +28,7 @@ jest.mock('@/server/lib/serverSideLogger', () => ({
 	},
 }));
 jest.mock('@aws-sdk/credential-providers');
-jest.mock('@aws-sdk/node-http-handler');
```

Comment:
`@aws-sdk/node-http-handler` has been deprecated, and instead told us to use `@smithy/node-http-handler` which we had installed already.